LayoutInspector/Classes/LayoutInspector.swift

```diff
@@ -120,3 +129,27 @@ extension LayoutInspector: LayoutInspectorPresenterDelegate {
         isInspecting = false
     }
 }
+
+extension LayoutInspector {
+	func addShake() {
+		guard motionManager.isAccelerometerAvailable else { return }
+		motionManager.accelerometerUpdateInterval = 0.1
+		motionManager.startAccelerometerUpdates(to: OperationQueue()) { [weak self] (data, error) in
+			guard let self = self else { return }
+			guard error == nil, let acceleration = data?.acceleration else {
+				self.motionManager.stopAccelerometerUpdates()
+				return
+			}
+			let accelerameter = sqrt( pow( acceleration.x , 2 ) + pow( acceleration.y , 2 )
+				+ pow( acceleration.z , 2) )
+			if accelerameter>2.3 {
```

Comment:
@miku1958 is 2.3 experimentally found value?

Reply:
yes